packages/mdc-snackbar/foundation.js

```diff
@@ -65,6 +66,14 @@ export default class MDCSnackbarFoundation extends MDCFoundation {
     this.adapter_.deregisterActionClickHandler(this.actionClickHandler_);
   }
 
+  dismissesOnAction() {
```

Comment:
Ups, how embarrassing. I added the getter and setter and adapted demos/snackbar.html.

Cheers, Michael